agrona/src/main/java/org/agrona/MarkFile.java

```diff
@@ -284,10 +284,22 @@ public int versionWeak()
 
     /**
      * Set timestamp field using an ordered put.
+     * <p>
+     * This method is identical to {@link #timestampRelease(long)} and that method is preferred.
      *
      * @param timestamp to be set.
      */
     public void timestampOrdered(final long timestamp)
+    {
+        timestampRelease(timestamp);
+    }
+
+    /**
+     * Set timestamp field using a release put.
+     *
+     * @param timestamp to be set.
+     */
+    public void timestampRelease(final long timestamp)
```

Comment:
the putLongOrdered below will be replaced by a putLongRelease as soon as the following PR is merged:

https://github.com/real-logic/agrona/pull/314